ui/tests/acceptance/auth/auth-login-test.js

```diff
@@ -54,6 +54,13 @@ module('Acceptance | auth login', function (hooks) {
     assert.dom(AUTH_FORM.selectMethod).hasValue('userpass');
   });
 
+  test('it pre-fills OIDC role if specified in query param', async function (assert) {
```

Comment:
Thank you for including test coverage! 😍  This should still pass with the changes I suggested